test/sqllogictest/copy_to.slt

```diff
@@ -0,0 +1,48 @@
+# Copyright Materialize, Inc. and contributors. All rights reserved.
+#
+# Use of this software is governed by the Business Source License
+# included in the LICENSE file at the root of this repository.
+#
+# As of the Change Date specified in that file, in accordance with
+# the Business Source License, use of this software will be governed
+# by the Apache License, Version 2.0.
+
+# Basic tests for `COPY ... TO` statements.
+
+mode cockroach
+
+# Start from a pristine state
+reset-server
+
+simple conn=mz_system,user=mz_system
+ALTER SYSTEM SET enable_copy_to_expr = on;
+----
+COMPLETE 0
+
+simple conn=mz_system,user=mz_system
+ALTER SYSTEM SET enable_aws_connection = on;
+----
+COMPLETE 0
+
+statement ok
+CREATE CONNECTION aws_conn TO AWS (ASSUME ROLE ARN 'assume-role', ASSUME ROLE SESSION NAME 'session-name');
+
+# Test correct tag is returned
+# TODO (mouli): currently this fails with actually: Values(["unexpected message from server"]), fix this
+simple conn=mz_system,user=mz_system
+COPY (VALUES (1, 2, 3)) TO 's3://path' WITH (AWS CONNECTION = aws_conn, FORMAT = 'csv', MAX FILE SIZE = '100MB')
+----
+COPY 2
```

Comment:
I am not entirely sure how to go about debugging this. Getting "unexpected message from server" back, but I don't see that string in our codebase?